cpp/src/arrow/adapters/orc/adapter.cc

```diff
@@ -720,11 +720,7 @@ class ArrowOutputStream : public liborc::OutputStream {
     return filename;
   }
 
-  void close() override {
-    if (!output_stream_.closed()) {
-      ORC_THROW_NOT_OK(output_stream_.Close());
-    }
-  }
+  void close() override {}
```

Comment:
There's nothing to do on the ORC side? No resources to release?

Reply:
No. What ORC does is writing the content. https://github.com/apache/orc/blob/7e12a8ccac81548fde6e5d0538da2e68c77df601/c%2B%2B/src/Writer.cc#L369